Library/Homebrew/language/python.rb

```diff
@@ -153,7 +153,8 @@ def needs_python?(python)
       def virtualenv_install_with_resources(options = {})
         python = options[:using]
         if python.nil?
-          wanted = %w[python python@2 python2 python3 python@3 pypy pypy3].select { |py| needs_python?(py) }
+          pythons = %w[python python@2 python2 python3 python@3 python@3.8 pypy pypy3]
```

Comment:
Worth adding in 3.9 for future proofing?

Reply:
It depends on the way we are going to handle python update in the future (by adding new formula or doing something else).
In general, I don't mind, will add something like:
```ruby
Formula.select { |f| f.name.start_with? "python@" }
```